### PR TITLE
Triggered notifications

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

File: packrat/scripts/notifier.js [hunk 1]
```diff
@@ -99,7 +101,7 @@ var notifier = function () {
   }
 
   function onClickGlobal(threadId, messageId, url, e) {
-    api.port.emit('set_global_read', {threadId: threadId, messageId: messageId});
+    api.port.emit('set_message_read', {threadId: threadId, messageId: messageId});
```
Comment (andrewconner) [inline packrat/scripts/notifier.js:118]: `set_global_read` wasn't actually handled, so this didn't work. `set_message_read` works as intended.

File: packrat/html/notify_box.html [hunk 2]
```diff
@@ -10,7 +10,12 @@
       <div class="kifi-on-page">On this page:</div>
       {{/subtitle}}
       {{#link}}<div class="kifi-notify-link">{{link}}</div>{{/link}}
+      {{#triggered}}
+      <div class="kifi-notify-triggered">asdasd{{#formatSnippet}}{{{contentHtml}}}{{/formatSnippet}}</div>
```
Comment (2is10) [inline packrat/html/notify_box.html:14]: `asdasd` ?

File: packrat/html/notify_box.html [hunk 3]
```diff
@@ -10,7 +10,12 @@
...
+      {{/triggered}}
+      {{^triggered}}
```
Comment (2is10) [inline packrat/html/notify_box.html:16]: I think the direction we ought to move is having per-category templates for these like we do for the Message Box.

File: packrat/scripts/notifier.js (hunk 1 above)
Comment (2is10) [inline packrat/scripts/notifier.js:118]: whoops, yes, I recently deprecated `"set_global_read"` and must have done it wrongly. good thing we haven’t sent any recently.

File: packrat/styles/keeper/notices.css [hunk 4]
```diff
@@ -311,6 +311,9 @@
 .kifi-notice-global>.kifi-notice-kicker::after {
   content: 'Announcement';
 }
+.kifi-notice-triggered.kifi-notice-triggered.kifi-notice-triggered>.kifi-notice-kicker::after {
```
Comment (2is10) [inline packrat/styles/keeper/notices.css:314]: We don’t need to repeat the class name three times. The build script does that.

**Comment by 2is10** [comment]
Nice work!

**Pull request merged**